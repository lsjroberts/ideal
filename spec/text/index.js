import { text } from 'ideal-ui';

export default text`hello`;
