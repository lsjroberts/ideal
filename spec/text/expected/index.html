<!DOCTYPE html>
<html>
  <head>
    <title>text</title>
  </head>
  <body>
    hello
  </body>
</html>
